pkg/selector/generic/mode.go

```diff
@@ -85,7 +86,7 @@ func FilterObjectsByMode(mode v1alpha1.SelectorMode, value string, count int) ([
 		}
 
 		percentage := getRandomNumber(maxPercentage + 1) // + 1 because Intn works with half open interval [0,n) and we want [0,n]
-		num := int(math.Floor(float64(count) * float64(percentage) / 100))
+		num := int(math.Ceil(float64(count) * float64(percentage) / 100))
```

Comment:
Should we remove the `maxPercentage+1` and update the comments?

Reply:
No. Consider the special situation where the `maxPercentage` is `100`, then the `percentage` will be `0-100` (as the `getRandomNumber` always returns the number in `[0,101)`), which is actually expected.

Reply:
@iguoyr PTAL